Sources/Tracking/CustomerIO.swift

```diff
@@ -88,16 +88,35 @@ public extension CustomerIOInstance {
         identify(identifier: identifier, body: body, onComplete: onComplete, jsonEncoder: jsonEncoder)
     }
 
+    func identify(
+        identifier: String,
+        body: [String: Any],
+        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void,
+        jsonEncoder: JSONEncoder? = nil
+    ) {
+        identify(identifier: identifier, body: StringAnyEncodable(body), onComplete: onComplete,
+                 jsonEncoder: jsonEncoder)
+    }
+
+    func track(
+        name: String,
+        data: [String: Any],
+        jsonEncoder: JSONEncoder?,
+        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void
```

Comment:
```suggestion
        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void,
        jsonEncoder: JSONEncoder? = nil
```

It's best to put parameters with default vales after all non-default params.